### PR TITLE
mandoc: 1.13.4 -> 1.14.4

### PR DESCRIPTION
**Pull request by bb010g**
Should I modify `meta.maintainers`? 1.14.1, the first update after 1.13.4, was in Feb 2017, so I'm not sure if @RamKromberg is still interested in maintaining mandoc.

[Changelog here](http://mandoc.bsd.lv/NEWS). It's late here, so I haven't really done testing outside of it successfully outputting HTML from mdoc, but I can do so if necessary.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] (N/A) Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/mandoc/default.nix [hunk 1]
```diff
@@ -26,8 +26,9 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    homepage = http://mdocml.bsd.lv/;
+    homepage = http://mandoc.bsd.lv/;
     description = "suite of tools compiling mdoc and man";
+    downloadPage = "http://mandoc.bsd.lv/snapshots/";
     license = licenses.bsd3;
     platforms = platforms.all;
     maintainers = with maintainers; [ ramkromberg ];
```
Comment (Mic92) [inline pkgs/tools/misc/mandoc/default.nix]: Do you want to add yourself as maintainer instead?

File: pkgs/tools/misc/mandoc/default.nix [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "mandoc-${version}";
-  version = "1.13.4";
+  version = "1.14.4";
 
   src = fetchurl {
-    url = "http://mdocml.bsd.lv/snapshots/mdocml-${version}.tar.gz";
-    sha256 = "1vz0g5nvjbz1ckrg9cn6ivlnb13bcl1r6nc4yzb7300qvfnw2m8a";
+    url = "http://mandoc.bsd.lv/snapshots/mandoc-${version}.tar.gz";
```
Comment (Mic92) [inline pkgs/tools/misc/mandoc/default.nix]: I get no dns records for this domain and the older version.

**Comment by Mic92** [comment]
@GrahamcOfBorg build mandoc

**Comment by bb010g** [comment]
It seems to be up currently? Also, switched over to https.

**Comment by Mic92** [comment]
@GrahamcOfBorg build mandoc

**Comment by Mic92** [comment]
You probably should try enabling nix's sandbox if you wonder why you have not seen this error in your build.

**Pull request merged**